assets/js/index.js

```diff
@@ -0,0 +1,906 @@
+		// The modal-target-string must match the string toggles use to target the modal
+		if (modal.dataset.modalTargetString) {
+			var modalTargetClass = modal.dataset.modalTargetString;
+			modalToggle = document.querySelector('*[data-toggle-target="' + modalTargetClass + '"]');
+		}
+
+		// If a modal toggle exists, trigger it so all of the toggle options are included
+		if (modalToggle) {
+			modalToggle.click();
+
+			// If one doesn't exist, just hide the modal
+		} else {
+			modal.classList.remove('active');
+		}
+	},
+
+}; // twentytwenty.coverModals
+
+/*	-----------------------------------------------------------------------------------------------
+	Dynamic Screen Height
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.dynamicScreenHeight = {
+
+	init: function () {
+		this.setScreenHeight();
+
+		setTimeout(function () {
+			this.setScreenHeight();
+		}.bind(this), 500);
+
+		window.addEventListener('resize', function () {
+			this.setScreenHeight();
+		}.bind(this));
+	},
+
+	setScreenHeight: function () {
+		var screenHeight = document.querySelector('.screen-height');
+		if (screenHeight) {
+			screenHeight.style.minHeight = window.innerHeight;
+		}
+	},
+
+}; // twentytwenty.dynamicScreenHeight
+
+/*	-----------------------------------------------------------------------------------------------
+	Focus Management
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.focusManagement = {
+
+	init: function () {
+		// If the visitor tabs out of the main menu, return focus to the navigation toggle
+		// Also, if the visitor tabs into a hidden element, move the focus to the element after the hidden element
+		this.focusLoop();
+	},
+
+	focusLoop: function () {
+		document.addEventListener('focusin', function (event) {
+			var element = event.target;
+			var menuModal = document.querySelector('.menu-modal');
+			var headerToggles = document.querySelector('.header-toggles');
+			var searchModal = document.querySelector('.search-modal');
+			if (menuModal.classList.contains('.active')) {
+				if (!menuModal.contains(element) && !headerToggles.contains(element)) {
+					document.querySelector('.nav-toggle').focus();
+				}
+			} else if (!searchModal.classList.contains('.active')) {
+				if (!searchModal.contains(element)) {
+					searchModal.querySelector('.search-field').focus();
+				}
+			}
+		});
+	},
+
+}; // twentytwenty.focusManagement
+
+/*	-----------------------------------------------------------------------------------------------
+	Intrinsic Ratio Embeds
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.intrinsicRatioVideos = {
+
+	init: function () {
+		this.makeFit();
+
+		window.addEventListener('fit-videos', function () {
+			this.makeFit();
+		}).bind(this);
```

Comment:
This should be `}.bind(this));`, same for the one below

Reply:
just fixed that  👍 thanks :)